File: Python/day2.py
contX=0
contY=0
contAux=0

with open('inputDay2.txt','r') as f:
    x_datos=[]
    y_datos=[]
    lineas=f.readlines()
    for linea in lineas:
        x,y=linea.split()
        x_datos.append(x)
        y_datos.append(int (y))
    
    for lineaAux in x_datos:
        if(lineaAux=='forward'):
            contX+=y_datos[contAux]
            contAux+=1
        if(lineaAux=='down'):
            contY+=y_datos[contAux]
            contAux+=1
        if(lineaAux=='up'):
            contY-=y_datos[contAux]
            contAux+=1
    print(contX*contY)

File: C++/day2.cpp
#include <fstream>
#include <iostream>
#include <string>

using namespace std;

//forward 416
//down 378
//up 206

int main(){
    ifstream file("inputDay2.txt");

    int contX=0;
    int contY=0;
    int contZ=0;
    string cadena;
    getline(file,cadena,' ');

    while(!file.eof()){
        if(cadena == "forward"){
            contX++;
        }
        else if(cadena == "down"){
            contY++;
        }
        else if(cadena == "up"){
            contZ++;
        }
        getline(file,cadena,' ');
    }

    cout << contX << endl;
    cout << contY << endl;
    cout << contZ << endl;
    cout << contX*contY << endl;
    file.close();
}